iot/mqtt_example/README.md

```diff
@@ -46,6 +46,7 @@ run the sample as:
         --registry_id=my-registry \
         --device_id=my-node-device \
         --private_key_file=../rsa_private.pem \
+        --cloud_region=us-west1 \
```

Comment:
Nit. I'm not 100% sure we're in the us-west1 region yet, can you change this to `us-central1`, `europe-west1`, or `asia-east1`?

Reply:
Did you verify?  I can modify.

Reply:
I did, `us-central1` is definitely supported, others I didn't list are not showing up for me.